File: Spang-PC_C-sharp/Spang.Core/Android/NetworkMessages/SensorEvent.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Spang.Core.Android.NetworkMessages
{
    class SensorEvent : IPhoneMessage
    {
        public int SensorID;
        public float[] SensorData;

        public SensorEvent(int id, float[] data)
        {
            // TODO: Complete member initialization
            this.SensorID = id;
            this.SensorData = data;
        }
    }
}

File: Spang Java Core/src/network/messages/SensorEvent.java
package network.messages;


public class SensorEvent implements IPhoneMessage{
	public final int SensorID;
	public final float[] SensorData;
	
    public SensorEvent(int id, float[] data)
    {
    	this.SensorID = id;
        this.SensorData = data;
    }
}
